src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  for (const key in robot) {
+    if (!(robot[key] in inversedRobot)) {
+      inversedRobot[robot[key]] = key;
+    } else {
+      return null;
+    }
```

Comment:
Такий варіант заввжди буде кращим

```suggestion
    if (robot[key] in inversedRobot) {
      return null;
    }
    
    inversedRobot[robot[key]] = key;
```